### ISSUE TITLE
Totals on Statistics page

### ISSUE THREAD

**Issue by kandykancan** [toopriddy/mytime#290]
There seems to be some trouble with the totals on the Statistics page. With the latest update, all of my previous months were marked as having made my time goal (I wish, unfortunately 🙃) and with the new "Placements" totals, if you add anything in manually, it isn't included under "Placements". There also seems to be some formatting issues in the address fields. Other than those things, I want to say that I really like this app and really appreciate your hard work on it. 🙂 Thank you very much!
Kandra Meyer

**Comment by toopriddy** [comment]
Hello Kandra, this is a bug about the Placements.  I sent apple an update 2 days ago and am waiting on them to approve the fix.

What formatting issues are you saying there are on the address fields?  (FYI there was also a mapped calls bug that got introduced in iOS 9 along with the 3.7.0 update).

I will need to close this bug as a duplicate of https://github.com/toopriddy/mytime/issues/287

**Comment by kandykancan** [comment]
Hi, I've attached some screen shots that will probably explain better than
I can. I also included one that shows the issue I have with the hours being
added wrong. Hope it helps!
~K~

**Comment by toopriddy** [comment]
I did not receive an attachment. Please email me directly at toopriddy@gmail.com

The hours goals is not a bug.  My time calculates the monthly average goal based on the yearly 840 goal. If you get more than 70 hours in previous months hen the remaining hours average per month to attain the 840 hour goal will be less than 70. Conversely if you get less than 70 hours a month the monthly goal will be more than 70

If you want a strict 70 our goal then you can go to  MyTime->More->Settings-> goals-> set a custom goal of 70 hours
